.circleci/config.yml

```diff
@@ -147,6 +147,13 @@ jobs:
     steps:
       - {attach_workspace: {at: .}}
       - {run: {name: 'Install and Test (check-failure)', command: 'install-and-test-ext check-failure'}}
+  test_check-pg-upgrade:
+    docker:
+      - {image: 'citus/pgupgradetester:latest'} 
```

Comment:
the other images have their pg versions in the image name. I think we should either put it in the image name or use a tag other then `latest` with the pg versions we are performing the upgrade for.

Reply:
If we put the pg versions in the image name, it would mean that we will have two different images if we want to have `(pg10->pg11)` and `(pg11->pg12)`. I thought that instead of doing that if we give the postgres versions as parameters we can just perform the same tests `(pg10->pg11)` and `(pg11->pg12)` in the same image.  So when we add `pg12` we can have something like:  
```yaml
- run: 
     name: 'Install and test postgres upgrade'
     command: 
          'install-and-test-ext --target check-upgrade --old-pg-version 10 --new-pg-version 11'
          'install-and-test-ext --target check-upgrade --old-pg-version 11 --new-pg-version 12'
```

if we create new images for each postgres combination then we will have more jobs. With citus upgrade tests we will have an additional job too. I think it makes sense to have separate images for different postgres versions when the job is running slow, because with 2 jobs it can be parallelized. 

What is the downside of having parameters?

Reply:
We could have one image that contains pg10, pg11, pg12, ... and perhaps even use it for all other tests as well.